docs/local_install_skaffold.md

```diff
@@ -1,51 +1,266 @@
+    - [Start Substra](#start-substra)
+    - [Stop Substra](#stop-substra)
+    - [Reset Substra](#reset-substra)
+  - [4. Login, password and urls](#4-login-password-and-urls)
+    - [Credentials and urls](#credentials-and-urls)
+    - [Browser extension](#browser-extension)
+    - [Substra CLI config & login](#substra-cli-config--login)
+  - [5. Troubleshooting](#5-troubleshooting)
+    - [Virtualization issues](#virtualization-issues)
+    - [Kubectl useful commands](#kubectl-useful-commands)
+    - [Minikube useful commands](#minikube-useful-commands)
+    - [Tiller](#tiller)
+    - [[WIP] Ongoing issues](#wip-ongoing-issues)
+  - [6. Need help?](#6-need-help)
+  - [7. Further resources](#7-further-resources)
+  - [8. Acknowledgements](#8-acknowledgements)
+
+___
+
+When everything is ready, you will be able to start Substra with:
 
-This guide will help you run the Substra platform on your machine in development mode, with a 2 organisations setup.
+```sh
+# Ubuntu only
+minikube start --cpus 6 --memory 8192 --disk-size 50g --kubernetes-version='v1.15.4'
+
+# In 3 different terminal windows, in this order:
+# In the repository hlf-k8s
+skaffold dev
+
+# In the repository susbtra-backend
+skaffold dev
+
+# In the repository substra-frontend
+skaffold dev
+```
+
+## 1. Substra Setup
+
+### General knowledge
+
+I order to install Substra, it is *recommended* to be comfortable with your package manager and to have some basic knowledge of Unix (Mac or Ubuntu/Debian) administration and network.
+
+### Hardware requirements
+
+> Please be cautious with the parameters passed to Kubernetes as it might try to use all the allocated resources without regards for your system!
+
+If you wish to comfortably run Substra, it is advised to have:
+
+- **50 GB of free space** as several images need to be pulled.
+- **8 GB of RAM** for Kubernetes.
+
+### Software requirements
+
+> Note: Please **always** refer to the package provider website before installing any software!
+> Note: This setup has been tested on Ubuntu 19.10, 18.04.3 & Linux Mint 19 Cinnamon (4.15.0-45-generic Kernel)
+
+Substra deployment is orchestrated by `Kubernetes` and `Minikube` is a great tool for your local Kubernetes deployments.
+
+`Helm` is a Kubernetes package manager that helps install and manage Kubernetes applications. `Tiller` runs inside the Kubernetes Cluster and manages releases of your charts. Helm has two parts : helm (client) & tiller (server). Charts are Helm packages that contains a package description (`Chart.yaml`) and one or more templates which contain Kubernetes manifest files. Charts can be stored locally or on a distant repository.
+
+`Skaffold` is a program working on top of the Kubernetes configuration that will operate the deployment for you. It relies on the `skaffold.yaml` files that you will find at the root of each repositories.
+
+#### [1. Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
```

Comment:
The title `Kubectl` seems incorrect with the content (at least for mac users).
 
What about refactoring the first 2 sections into:

1. Running Kubernetes locally
- Ubuntu: minikube
- Mac: Docker for mac

2. Installing Kubectl
- Ubuntu:  `apt-get`
- Mac: `brew install kubectl`